app/src/main/java/org/fossasia/openevent/general/SplashActivity.kt

```diff
@@ -0,0 +1,14 @@
+package org.fossasia.openevent.general
+
+import android.content.Intent
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        startActivity(Intent(this, MainActivity::class.java))
+        finish()
+    }
+}
```

Comment:
No new Activity should be created

Reply:
https://plus.google.com/+AndroidDevelopers/posts/Z1Wwainpjhd

Reply:
Thank you. I will follow the documentation.